eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -468,7 +468,7 @@ jobs:
             if [[ '${{ parameters.enablePoison }}' == 'True' ]]; then
               customBuildArgs="$customBuildArgs --poison"
             fi
-            customBuildArgs="$customBuildArgs --source-only /p:SmokeTestsWarnOnSdkContentDiffs=true /p:SmokeTestsExcludeOmniSharpTests=${{ parameters.excludeOmniSharpTests }}"
```

Comment:
Was `/p:SmokeTestsWarnOnSdkContentDiffs=true` removed intentionally?

Reply:
Yes.  I believe this is a remnant from when we ran the sdk content tests as part of the main pipeline.  Now that they are run separately this flag does nothing as it is only reference in the sdk content tests.  Instead of renaming I decided to clean it up.